src/pages/Layout.jsx

```diff
@@ -27,21 +25,9 @@ const Layout = () => {
       <Box component="main" sx={{ flexGrow: 1, p: 3 }}>
         <Toolbar variant="tall" />
         <Outlet />
-        <Box
-          sx={{
-            display: "flex",
-            justifyContent: "center",
-            alignItems: "center",
-          }}
-        >
-          Powered by
-          <Link to={"https://www.nextbyteweb.com/"} target="_blank">
-            <img src="./logo-black.webp" alt="NextByte" height={30} />
-          </Link>
-        </Box>
+        <Footer />
```

Comment:
It looks like the Footer should be outside this current Box. It looks like it should be in between line 29 and 30

Reply:
Yields the same results but you have to change the flex-direction of the enclosing box.
I have done that.